Nodejs/Product/Npm/Npm.csproj

```diff
@@ -163,6 +163,26 @@
     </EmbeddedResource>
   </ItemGroup>
   <ItemGroup>
+    <EmbeddedResource Include="Resources.cs.resx" />
```

Comment:
Does this not need to be dependent on Dev15 like the other projects localized items?